src/sentry/integrations/slack/utils.py

```diff
@@ -378,6 +379,32 @@ def get_channel_id(organization, integration, name, use_async_lookup=False):
     return get_channel_id_with_timeout(integration, name, timeout)
 
 
+def is_valid_channel_id(name, organization, integration_id, use_async_lookup, input_channel_id):
```

Comment:
Hmm yeah when I was writing https://github.com/getsentry/sentry/pull/24314 I initially wasn't expecting the channel name to be provided as well, but that's what https://github.com/getsentry/sentry/pull/20338 does so I wanted to keep it consistent so it's an additional field rather than replacing name with the id.

Reply:
I'll leave that up to your team to decide, but from my point of view if I was a user, and I slightly typoed a channel name but had provided an ID I'd probably be annoyed if it failed until I got both right.